packages/server/patches/http-proxy+1.18.1.patch

```diff
@@ -0,0 +1,2584 @@
+diff --git a/node_modules/http-proxy/.auto-changelog b/node_modules/http-proxy/.auto-changelog
+old mode 100644
+new mode 100755
+diff --git a/node_modules/http-proxy/.gitattributes b/node_modules/http-proxy/.gitattributes
+deleted file mode 100644
+index 1a6bd45..0000000
+--- a/node_modules/http-proxy/.gitattributes
++++ /dev/null
+@@ -1 +0,0 @@
+-package-lock.json binary
+diff --git a/node_modules/http-proxy/CHANGELOG.md b/node_modules/http-proxy/CHANGELOG.md
+deleted file mode 100644
+index 6c80d84..0000000
+--- a/node_modules/http-proxy/CHANGELOG.md
++++ /dev/null
+@@ -1,1872 +0,0 @@
+-# Changelog
+-
+-All notable changes to this project will be documented in this file.
+-
+-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+-
+-Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+-
+-## [v1.18.1](https://github.com/http-party/node-http-proxy/compare/1.18.0...v1.18.1) - 2020-05-17
+-
+-### Merged
```

Comment:
@AtofStryker Is this patch supposed to have all of this information in here? I though it's typically a few lines.

Reply:
I've been trying to figure this out and I think its out friend `.yarnclean` https://github.com/cypress-io/cypress/blob/develop/.yarnclean#L53. going to turn it off to develop the patch then reenable it

Reply:
@jennifer-shehane it was the `yarnclean`. should be fixed now

Reply:
Nice catch